reference/tax_properties.v3.yml

```diff
@@ -0,0 +1,306 @@
+      parameters:
+        - name: store_hash
+          in: path
+          required: true
+          type: string
+        - name: 'product_id:in'
+          in: query
+          required: true
+          type: string
+          description: 'Comma-separated list of Product IDs. Example ''12,34,56'''
+      responses:
+        '204':
+          description: No Content
+        '400':
+          description: Request parameters invalid
+definitions:
+  Property:
+    type: object
+    properties:
+      id:
+        type: integer
+        description: An internal identifier used by other operations.
+        example: 1
+        readOnly: true
+      code:
+        type: string
+        description: The unique string programmatically references this unique tax property by the tax provider. Merchants should liaise with their tax provider to explore supported values and functions.
+        example: A-123456789
+      display_name:
+        type: string
+        description: The human-readable name for this property. This string will be displayed on the Products screen as a field label.
+        example: Example Tax Property 1
+      description:
+        type: string
+        description: Optional. Additional detail about this tax property may help guide merchants. This string will be displayed on the Products screen as a tooltip associated with the relevant field.
+        example: Food Industry
+      created_at:
+        type: string
+        description: Datetime.
+        readOnly: true
+      updated_at:
+        type: string
+        description: Datetime.
+        readOnly: true
+  ProductTaxProperty:
+    type: object
+    description: A collection of tax property values associated with a product.
+    properties:
+      product_id:
+        type: integer
+        description: A reference to the product that the product tax properties are associated with.
+        example: 1
+      tax_properties:
+        type: object
+        description: A simple key-value pairing. The tax property must be defined to associate a value. These values will be sent to the active tax provider during Tax Provider API operations whenever the associated product is included in the operation.
+        properties:
+          <example tax property code 1>:
+            type: string
+            example: '405'
+          <example tax property code 2>:
+            type: string
+            example: '111'
+          <example tax property code 3>:
+            type: string
+            example: '250'
+    required:
+      - product_id
+      - tax_properties
+securityDefinitions:
+  API Key:
+    type: apiKey
+    name: X-Auth-Token
+    in: header
+    description: |-
+      ### OAuth Scopes
+      |  **UI Name** | **Permission** | **Parameter** |
+      | --- | --- | --- |
+      |  Information & Settings | modify | `store_v2_information` |
+      |  Information & Settings | read-only | `store_v2_information_read_only` |
```

Comment:
cc @theromulans here. I added `Info & Settings` as the OAuth Scope because the other tax endpoints had this scope, but was hoping for a quick confirmation

Reply:
Yes that's correct, thanks @bc-andreadao!